diesel/src/sqlite/connection/raw.rs

```diff
@@ -24,8 +24,10 @@ impl RawConnection {
     pub fn establish(database_url: &str) -> ConnectionResult<Self> {
         let mut conn_pointer = ptr::null_mut();
         let database_url = CString::new(database_url.trim_start_matches("sqlite://"))?;
```

Comment:
Just a random idea here: Should we treat `sqlite://URI` the same as `file://URI`? If so we should change the code in such a way that we replace `sqlite://` at the beginning of a connection URI with `file://` instead of just removing that part.

Reply:
Makes sense to me. Done.